### ISSUE TITLE
separatedDefinitionsEnabled feature is broken or useless 

### ISSUE THREAD

**Issue by alkemann** [Swagger2Markup/swagger2markup#258]
If I enable the *separatedDefinitionsEnabled* feature I do get my definition objects created in individual files, but it still creates the "parent" page, which becomes just a list of H3 tags which anchor links to itself, when they obviously should link to individual files where the content is actually found.

Alternatively, who can I disable the generation of the definitions document, which is kinda pointless onces the content is moved out of it.

Swagger2Markup version: 1.3.1 & 1.3.2-SNAPSHOT

**Comment by RobWin** [comment]
Hi,
true its pointless. Would you like to contribute a fix? I'm in vacation and afterwards busy at work. But I could merge a PR an create a new release.

**Comment by alkemann** [comment]
Yes I have the code running locally in Intellj but I cant figure out how to fix this. If you could give me some pointer in the right direction that would be great.

This isnt the only case of these broken links either. As far as I can see the default behavior is that the paths links to definitions also just link to an anchor in the same document.

**Comment by RobWin** [comment]
There is a swagger2markup configuration parameter which enables inter-document cross references.

**Comment by RobWin** [comment]
swagger2markup.interDocumentCrossReferencesEnabled

**Comment by alkemann** [comment]
I pressume you mean this: swagger2markup.interDocumentCrossReferencesEnabled

Why is this a config and how does it make sense to have it as false by default?

**Comment by alkemann** [comment]
Yes this did fix my issues. I would argue that the default setting for this option should be reversed, or even just removed at all as why someone wouldnt want their anchors to work I dont quite understand.

**Comment by alkemann** [comment]
Is there a way to just drop the list file when enabling the separeted<x>enabled then becomes the remaining issue.

**Comment by RobWin** [comment]
Yes, you are right. I have to check the code again. I can't remember why it's false by default. It's some time ago since I programmed it.

**Comment by alkemann** [comment]
I can always just delete them after generation I guess.

So, of lower priority, this issue becomes a suggestion to reverse the default and to add a feature of optionally dropping the main documents, security, paths, definitions and/or overview

**Comment by RobWin** [comment]
Currently there is no way to skip the generation of the definitions file.

**Comment by alkemann** [comment]
most of these cross document links seems to work now, but the one that is still off is the link to security document from paths endpoints.

**Comment by MaksimMyshkin** [comment]
Hi. I think separatedDefinitionsEnabled is false by default because in most cases you will generate single file (pdf or html) from markdown/asciidoc sources. In this case you need that references work within one file.
